sdk/eventgrid/azure-eventgrid/azure/eventgrid/_signature_credential_policy.py

```diff
@@ -4,10 +4,15 @@
 # license information.
 # -------------------------------------------------------------------------
 
+from typing import Any, TYPE_CHECKING
```

Comment:
is Any used?

Reply:
yes - on line 24

Reply:
Edit: answered my own question, was already in type signatures elsewhere.  SGTM.